src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
Regarding rospack, I have patches as well (you can find them [here](https://anonscm.debian.org/cgit/debian-science/packages/ros/ros-rospack.git/tree/debian/patches)) and I will submit them once we found a solution over here, as they are quite similar.